tutorials/parachain-to-polkadot-vault/0-intro.mdx

```diff
@@ -0,0 +1,35 @@
+---
+tags:
+  - metadata
+  - wallet
+  - qr
```

Comment:
add parachain maybe instead of qr? it will later be possible to filter tutorials by tags and i think "parachain" would make more sense than qr

Reply:
Agreed